src/cf-buttons/src/atoms/buttons.less

```diff
@@ -169,4 +170,14 @@
       padding-bottom: unit( @btn-v-padding-modifier-ie * 15px / @btn__super-font-size, em );
     }
   }
+
+  //
+  // Full width button on x-small screens
+  //
+  &__full-on-xsmall {
```

Comment:
I went back and forth. We currently use `-on-small` elsewhere. If you'd rather the vars we should change those too.